webapp/views.py

```diff
@@ -36,7 +36,7 @@
 def get_cve(cve_id, **kwargs):
     show_hidden = kwargs.get("show_hidden", False)
 
-    cve = db_session.query(CVE).filter(CVE.id == cve_id).one_or_none()
+    cve = db_session.query(CVE).get(cve_id.upper())
```

Comment:
If you search for "CVE.id ==", I think you'll find at least one occurrence of this same conditional, which I'd imagine has the same bug. Could you check?

Reply:
4 occurrences, yes. Should I change those too?